libraries/k8sobject.rb

```diff
@@ -20,49 +21,60 @@ class K8sObject < K8sResourceBase
     def initialize(opts = {})
       # Call the parent class constructor
       super(opts)
- 
+
       @objapi = opts[:api] if opts[:api] ||= 'v1'
       @objtype = opts[:type] if opts[:type] ||= nil
       @objname = opts[:name] if opts[:name] ||= nil
       @objnamespace = opts[:namespace] if opts[:namespace] ||= nil
 
-      if !@objnamespace.nil?
-        @display_name = "#{@objnamespace}/#{@objname}"
-      else
-        @display_name = "#{@objname}"
-      end  
+      @display_name = if !@objnamespace.nil?
```

Comment:
Minor suggestion: can we change`if !@objnamespace.nil?` to `unless @objnamespace.nil?`